docs/v3/core/v3.0.rst

```diff
@@ -597,30 +597,63 @@ mandatory names:
     Provides an element value to use for uninitialised portions of the
     Zarr array.
 
-    If the data type of the Zarr array is Boolean then the value must
-    be the literal ``false`` or ``true``. If the data type is one of
-    the integer data types defined in this specification, then the
-    value must be a number with no fraction or exponent part and must
-    be within the range of the data type.
+    The permitted values depend on the data type:
 
-    For any data type, the ``fill_value`` is required. The literal
-    ``null`` is not permitted. The fill value needs to be defined
-    so that the data is independent of implementation details. Internally
-    implementations may provide a default ``fill_value``, but that must
-    be converted to a fixed value in the stored metadata.
+    ``bool``
+      The value must be a JSON boolean (``false`` or ``true``).
 
-    If the ``data_type`` of an array is defined in a ``data_type`` extension,
-    then said extension is responsible for interpreting the value of
-    ``fill_value`` and returning a suitable type that can be used.
+    Integers (``{uint,int}{8,16,32,64}``)
+      The value must be a JSON number with no fraction or exponent part that is
+      within the representable range of the data type.
 
-    For core data types for which fill values are not permitted in JSON or
-    for which decimal representation could be lossy, a string representing of
-    the binary (starting with ``0b``) or hexadecimal value (starting with
-    ``0x``) is accepted. This string must include all leading or trailing
-    zeroes necessary to match the given type size. The string values ``"NaN"``,
-    ``"+Infinity"`` and ``"-Infinity"`` are also understood for floating point
-    data types.
+    IEEE 754 floating point numbers (``float{16,32,64}``)
+      The value may be either:
 
+      - A JSON number, that will be rounded to the nearest representable value.
+
+      - A JSON string of the form:
+
+        - ``"+Infinity"``, denoting positive infinity;
```

Comment:
Note that Zarr V2 uses "Infinity" without leading '+' . Is the discrepancy ZarrV2 vs ZarrV3 intended ?

Reply:
That wasn't intentional --- I changed it back.